### PR TITLE
Added files for Session 2 -23rd May'21

### PR DESCRIPTION
**Pull request by chaitak-gorai**

### CODE REVIEW

**Comment by Dsantra92** [comment]
@chaitak-gorai thank you for the pr. I just want to point out few things in general.
- Don't need to put up the notebook for now. It needs to be cleaned and curated.
- Add your name to the list of contributors in the main readme.

File: Study_Session_2/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Session 2
+
+**Date of session** : 23rd March 2021
+
+Objectives of the session:
+
```
Comment (Dsantra92) [inline Study_Session_2/README.md:6]: Can you elaborate on the things mentioned here? I mean just a small paragraph containing the summary of what we covered would look nice.

File: Study_Session_2/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- Loop
+
+2.Introduction to Probability with Python
+
+- random module
+- nbiased and biased coin toss
+- Graph
+
+## Resources
+
```
Comment (Dsantra92) [inline Study_Session_2/README.md:92]: Notebook link not required for now.

Comment (chaitak-gorai) [reply Study_Session_2/README.md:92]: Ok, I have updated it.

File: Study_Session_2/README.md (hunk 1 above)
Comment (chaitak-gorai) [reply Study_Session_2/README.md:6]: Sure, I am adding it.

**Comment by Dsantra92** [comment]
Thanks, Chaitak. I will be adding my own commits to this branch and then we can merge these commits as a whole.

**Comment by Dsantra92** [comment]
Some images that I found safe to store here.
[Always has been](https://user-images.githubusercontent.com/55111154/119298887-8dbe7280-bc7b-11eb-9d09-ad95f94dd177.png)

[Tuesday 25 May 2021 11:21:06 PM IST](https://user-images.githubusercontent.com/55111154/119544982-e68f2780-bdaf-11eb-8044-f67acf5c3ccb.png)

**Comment by chaitak-gorai** [comment]
> Some images that I found safe to store here.
> [Always has been](https://user-images.githubusercontent.com/55111154/119298887-8dbe7280-bc7b-11eb-9d09-ad95f94dd177.png)

Oops :)

**Comment by Dsantra92** [comment]
Thanks a lot, @chaitak-gorai !!!

**Comment by chaitak-gorai** [comment]
> Thanks a lot, @chaitak-gorai !!!
> 
> 

Happy to contribute :)

**Pull request merged**